rotkehlchen/inquirer.py

```diff
@@ -690,7 +772,7 @@ def find_yearn_price(
             return None
 
         underlying_token = EvmToken(ethaddress_to_identifier(maybe_underlying_token[0].address))
-        underlying_token_price = self.find_usd_price(underlying_token)
+        underlying_token_price, _ = self.find_usd_price_and_oracles(underlying_token)
```

Comment:
can't you use the wrapper you made?

Reply:
Yes, this is the terrible idea I had before creating the wrapper